packages/sanity/src/structure/panes/document/documentPanel/banners/DraftLiveEditBanner.tsx

```diff
@@ -0,0 +1,74 @@
+import {ErrorOutlineIcon} from '@sanity/icons'
+import {useTelemetry} from '@sanity/telemetry/react'
+import {Flex, Text} from '@sanity/ui'
+import {useCallback, useEffect, useState} from 'react'
+import {isDraftId, useDocumentOperation, useTranslation} from 'sanity'
+
+import {Button} from '../../../../../ui-components'
+import {structureLocaleNamespace} from '../../../../i18n'
+import {useDocumentPane} from '../../useDocumentPane'
+import {
+  DiscardedLiveEditDraft,
+  PublishedLiveEditDraft,
+} from './__telemetry__/DraftLiveEditBanner.telemetry'
+import {Banner} from './Banner'
+
+export function DraftLiveEditBanner(): JSX.Element | null {
+  const {displayed, documentId} = useDocumentPane()
```

Comment:
Could we pass these in via props instead? We already need this info in the consuming end, so it should add any extra work (and removes a dependency on the document pane)

Reply:
That is true :) 
Done!